test/framework/src/main/java/org/elasticsearch/entitlement/runtime/policy/TestPolicyManager.java

```diff
@@ -97,6 +97,10 @@ public final void clearModuleEntitlementsCache() {
 
     @Override
     protected boolean isTrustedSystemClass(Class<?> requestingClass) {
+        if (requestingClass.getPackageName().startsWith("sun.java2d")) {
```

Comment:
Same here, this would become: `excludedSystemModulesByPackage.contains(requestingClass.getPackageName())`